### PR TITLE
Hotfix/4.5.2

### PR DESCRIPTION
**Pull request by nsoblath**
## New Features
* Adds a dev docker image

## Fixes
* Build now works with dripline-cpp as an external dependency.  Uses dl-cpp v2.6.2.

## Testing
* Builds locally in docker
* Builds on Travis
* TODO: dl-cpp applications print debug text
* TODO: dl-cpp test applications and examples are present

## Prior to merging for releases:
- [x] update the project's version in the top-level `CMakeLists.txt` file
- [x] update the `appVersion` to be the new container image tag version in `chart/Chart.yaml`

### CODE REVIEW

**Comment by laroque** [comment]
I think this looks pretty good, pending the merge of dripline-cpp 2.6.2 and successful CI jobs.

**Review by laroque — APPROVED** [review]

**Comment by nsoblath** [comment]
@laroque I believe this PR is ready now.  The GitHub Actions build stuff is new, so you should take a look before we merge.

**Pull request closed without merging**
